addons/mail/static/src/core/public_web/discuss.js

```diff
@@ -59,7 +59,7 @@ export class Discuss extends Component {
         this.orm = useService("orm");
         this.effect = useService("effect");
         this.ui = useState(useService("ui"));
-        useChildSubEnv({
+        useSubEnv({
```

Comment:
This change doesn't seams related to the PR, why change to `useSubEnv`?

Reply:
Because `discuss.inDiscussApp` was not available on the discuss component, which is stupid. IMO `useChildSubEnv()` looks very specific compared to `useSubEnv()` and it doesn't seem that the use of `ChildSubEnv` here was motivated.